Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1429,9 +1430,11 @@ static void addUserInstalledFontURLs(NSString *path, Vector<URL>& fontURLs)
     RetainPtr enumerator = [NSFileManager.defaultManager enumeratorAtPath:path];
 
     for (NSString *font in enumerator.get()) {
-        if ([font hasSuffix:@"ttf"] || [font hasSuffix:@"ttc"] || [font hasSuffix:@"otf"]) {
-            RetainPtr fontsPath = [path stringByAppendingPathComponent:font];
-            auto fontURL = URL::fileURLWithFileSystemPath(String(fontsPath.get()));
+        RetainPtr fontsPath = [path stringByAppendingPathComponent:font];
+        auto fontURL = URL::fileURLWithFileSystemPath(String(fontsPath.get()));
+        NSURL *nsFontURL = fontURL;
```

Comment:
I think that you can replace this with `NSURL *nsFontURL = [NSURL fileURLWithPath:[path stringByAppendingPathComponent:font]]`.